desktop/release-notes.md

```diff
@@ -23,6 +23,92 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+
+### Upgrades
+- [Docker Compose v2.7.0](https://github.com/docker/compose/releases/tag/v2.7.0)
+- [Docker Compose "Cloud Integrations" v1.0.28](https://github.com/docker/compose-cli/releases/tag/v1.0.28)
+- [Kubernetes v1.24.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.24.2)
+- [Go 1.18.4](https://github.com/golang/go/releases/tag/go1.18.4)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Adds the Container / Compose icon as well as the exposed port(s) / exit code to the Containers screen.
+- Updated Docker theme palette colours values to match design system.
+- Improved an error message from `docker login` if Registry Access Management is blocking the Docker engine's access to Docker Hub.
+- Increase throughput between the Host and Docker, for example increasing performance of "docker cp".
+- Collecting diagnostics takes less time to complete.
+- Selecting/unselecting a compose app on the containers overview will select/unselect all its containers.
+- Show tag names on the container overview image column.
+- Added search decorations to the terminal's scrollbar so that matches outside the viewport are visible.
+- Fixed issue on search which doesn't work well on containers page [docker/for-win#12828](https://github.com/docker/for-win/issues/12828).
+- Fixed issue on infinite loading on volume screen [docker/for-win#12789](https://github.com/docker/for-win/issues/12789).
+- Fixed a problem in the Container UI where resizing or hiding columns didn't work. Fixed [docker/for-mac#6391](https://github.com/docker/for-mac/issues/6391).
+- Fixed a bug where the state of installing, updating, or uninstalling multiple extensions at once was lost when leaving the Marketplace screen.
+- Fixed issue where compose version in the about page would only get updated from v2 to v1 after restart.
+- Fixed an issue where user cannot not see log view because their underlying hardware didn't support WebGL2 rendering [docker/for-win#12825](https://github.com/docker/for-win/issues/12825).
+- Added setting under Settings -> General to override the OS's light mode / dark mode. Addresses [docker/for-win#12747](https://github.com/docker/for-win/issues/12747).
+- Fixed a bug where the UI for Containers and Images got out of sync.
+- Fix a startup race when the experimental virtualization framework is enabled.
+
+#### For Mac
+
+- Fixed an issue executing Compose commands from the UI. Fixes [docker/for-mac#6400](https://github.com/docker/for-mac/issues/6400).
+
+#### For Windows
+
+- Fixed resizing issue on Windows. Fixes [docker/for-win#12816](https://github.com/docker/for-win/issues/12816).
+- Send HTTP and HTTPS traffic from containers via the proxy.
+- Fixed an issue doing actions in the UI with Compose apps that were created from a WSL distro. Fixes [docker/for-win#12806](https://github.com/docker/for-win/issues/12806).
+
+
+#### For Linux
+
+- Fixed bug related to setting up file shares with spaces in their path on Linux.
```

Comment:
Isn't stating "on Linux" duplicate here due to being in the "For Linux" section ?